src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1701,6 +1701,7 @@ void primitive_inst::do_runtime_in_place_concat() {
         }
     }
     GPU_DEBUG_TRACE_DETAIL << "[In place concat] update shape for " << concat_inst->id() << std::endl;
+    auto prev_concat_layout = concat_inst->_impl_params->get_output_layout();
```

Comment:
nit: what about placing it close to the conditional statement that uses this variable?